src/lsp/client/downloader.ts

```diff
@@ -36,7 +36,7 @@ export function getClojureLspPath(
 ): string {
   let name = getArtifactDownloadName(platform, arch);
   if (path.extname(name).toLowerCase() !== '.jar') {
-    name = arch === 'win32' ? 'clojure-lsp.exe' : 'clojure-lsp';
+    name = platform === 'win32' ? 'clojure-lsp.exe' : 'clojure-lsp';
```

Comment:
I'm thinking that since this has worked before, maybe we need to consider earlier versions of VS Code? Is it possible for you to try your fix with an earlier version of VS Code on the machine where you can reproduce the problem Previous versions can be found in the sidebar here: https://code.visualstudio.com/updates/v1_81

Reply:
Good idea. I have yet to find the root cause of the problem, I will do some more investigating as well as trying previous VS Code versions tomorrow!